pkg/private/tar/build_tar.py

```diff
@@ -309,7 +309,10 @@ def add_manifest_entry(self, entry_list, file_attributes):
     elif entry.entry_type == manifest.ENTRY_IS_EMPTY_FILE:
       self.add_empty_file(entry.dest, **attrs)
     else:
-      self.add_file(entry.src, entry.dest, **attrs)
+      if os.path.isdir(entry.src):
```

Comment:
This is not the place we should fix it.
For correctness, it needs to be in pkg.bzl, where we test that the input file has `is_directory` set.  But we are already doing that.